packages/react-components/scripts/build.js

```diff
@@ -8,7 +8,7 @@ const path = require('path');
 
 console.log('Starting component-library build');
 console.log('Cleaning old build');
-rimraf.sync('./*.js');
+rimraf.sync('./[.]*!(index).js');
```

Comment:
We don't want the build script to delete that `index.js` file that exports all of the components.